interface/ceed-vector.c

```diff
@@ -288,6 +336,7 @@ int CeedVectorSetValue(CeedVector vec, CeedScalar value) {
 
   if (vec->SetValue) {
     CeedCall(vec->SetValue(vec, value));
+    vec->state += 2;
```

Comment:
Out of curiosity, why is `state` incremented by 2? Is the first bit reserved for something? (read/write access to the array)?

Reply:
Nevermind, I see that's exactly what it's used for in `CeedVectorSetValueStrided`. haha

Reply:
An odd state indicates write access has been granted. Jumping by 2 means write access was granted and restored so array contents must be assumed to be different.